executor/runtime/docker/docker_isolate.go

```diff
@@ -82,6 +87,11 @@ func workloadIsolated(ctx context.Context, taskID, host string) bool {
 		return true
 	}
 
+	if resp.StatusCode == 404 {
+		// Only log unexpected status codes below
+		return false
+	}
+
 	logrus.WithFields(map[string]interface{}{
 		"statusCode": resp.StatusCode,
 		"status":     resp.Status,
```

Comment:
Can you make it say `Titus isloate didn't return 200 after N tries?` ?

Reply:
Yeah, I added the line about tries above.  I don't think it's important to log what try an unexpected request occurred on - only what was returned.